### PR TITLE
[webkitscmpy] Allow repositories to define custom setup commands

### PR DESCRIPTION
**Pull request by JonWBedard**
#### 5a24f534edd68bf73e65b3d2b8c14ff0748bcc88
<pre>
[webkitscmpy] Allow repositories to define custom setup commands
<a href="https://bugs.webkit.org/show_bug.cgi?id=231345">https://bugs.webkit.org/show_bug.cgi?id=231345</a>
&lt;rdar://problem/83960249 &gt;

Reviewed by Dewei Zhu.

* Tools/Scripts/git-webkit: Define changelog conflict resolver.
* Tools/Scripts/libraries/webkitscmpy/setup.py: Add inspect2 as dependency, bump version.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py: Ditto.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/__init__.py:
(main): Attempt to resolve additional_setup function.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/setup.py:
(Setup.github): Invoke additional_setup function, if it exists.
(Setup.git): Ditto.
</pre>

### CODE REVIEW

File: Tools/Scripts/git-webkit [hunk 1]
```diff
@@ -45,10 +46,30 @@ def is_webkit_filter(to_return):
...
+    log.warning('Setting merging behavior for changelogs...')
+    if run([
+        repository.executable(),
+        'config', 'merge.changelog.driver',
+        'perl {} --fix-merged --merge-driver %O %A %B'.format(os.path.join(
+            os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(__file__)))),
+            'OpenSource', 'Tools', 'Scripts', 'resolve-ChangeLogs',
+        )),
+    ], capture_output=True, cwd=repository.root_path).returncode:
+        sys.stderr.write('Failed to set the git merging behaivor for changelogs...\n')
```
Comment (dewei-zhu) [inline Tools/Scripts/git-webkit:62]: Is using `sys.stderr` instead of `log.error` intended?

**Review by dewei-zhu — APPROVED** [review]
r=me

File: Tools/Scripts/git-webkit (hunk 1 above)
Comment (JonWBedard) [reply Tools/Scripts/git-webkit:62]: Yes, I don't want someone to be able to disable that error message with `-q`

**Comment by JonWBedard** [comment]
Landed https://commits.webkit.org/243086@main (d7527b8893ccbd0df0c86648fa41dce641ee4201)

**Pull request closed without merging**